types/knex.d.ts

```diff
@@ -422,9 +425,11 @@ declare namespace Knex {
     (columnNames: ColumnName[]): QueryBuilder;
   }
 
+  type RawBinding = (Value | QueryBuilder)[];
+
   interface RawQueryBuilder {
-    (sql: string, ...bindings: (Value | QueryBuilder)[]): QueryBuilder;
-    (sql: string, bindings: (Value | QueryBuilder)[] | ValueMap): QueryBuilder;
+    (sql: string, ...bindings: RawBinding): QueryBuilder;
```

Comment:
I can understand RawBinding being added, but aren't Value and QueryBuilder valid params as well? Shouldn't they be also kept?

Reply:
I agree, I see no need to remove them.